.gitignore

```diff
@@ -5,3 +5,4 @@
 /my-app.*
 /var/
 /vendor/
+.idea/
```

Comment:
Could you revert that? It must be in your global git ignore 

https://stackoverflow.com/a/7335487/685587